pkgs/by-name/go/go-car/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  libpcap,
+}:
+
+buildGoModule rec {
+  pname = "go-car";
+  version = "2.14.3";
+
+  src = fetchFromGitHub {
+    owner = "ipld";
+    repo = "go-car";
+    rev = "v${version}";
+    hash = "sha256-I1CVOoENQ5Qecl+8o4jbQYvXDYTPsU8QRcxCLwkJdng=";
+  };
+
+  modRoot = "cmd";
+  subPackages = [ "car" ];
+
+  vendorHash = "sha256-GO8PR20xHEybiFW/ukRZh5Rlt/Y/iJfMcnd1IVQ9Rtk=";
+
+  buildInputs = [
+    libpcap
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "A content addressable archive utility";
```

```suggestion
    description = "Content addressable archive utility";
```
Not start with the definite or an indefinite article.